resources/sdk-libraries.md

```diff
@@ -0,0 +1,52 @@
+---
+title: SDKs and Libraries
+sidebar:
+  navigation:
+  - title: Resources
+    items:
+    - url: /resources/
+      title: Introduction
+    - url: /resources/sdk-libraries
+      title: SDKs and Libraries
+    - url: /resources/test-data
+      title: Test Data
+    - url: /resources/demoshop
+      title: Demoshop
+    - url: /resources/development-guidelines
+      title: Open Source Development Guidelines
+    - url: /resources/release-notes
+      title: Release Notes
+    - url: /resources/terminology
+      title: Terminology
+    - url: /resources/data-protection
+      title: Data Protection
+    - url: /resources/public-migration-key
+      title: Public Migration Key
+---
+
+
+{% include jumbotron.html body="We have multiple Open Source-based SDKs and Libraries to use with Swedbank Pay APIs" %}
+
+## Swedbank Pay Official SDKs
+
+{:.table .table-striped}
+| Language        | Repository                   | Version           |
+| :-------------- | :--------------------------- | :---------------- |
+| [PHP][swedbank-pay-sdk-php] | [SwedbankPay/swedbank-pay-sdk-php][swedbank-pay-sdk-php] | [![](https://poser.pugx.org/swedbank-pay/swedbank-pay-sdk-php/version)](https://packagist.org/packages/swedbank-pay/swedbank-pay-sdk-php) |
+| [.NET][swedbank-pay-sdk-dotnet] | [SwedbankPay/swedbank-pay-sdk-dotnet][swedbank-pay-sdk-dotnet] | Beta |
+| [iOS][swedbank-pay-sdk-ios] | [SwedbankPay/swedbank-pay-sdk-ios][swedbank-pay-sdk-ios] | Beta |
+| [Android][swedbank-pay-sdk-android] | [SwedbankPay/swedbank-pay-sdk-android][swedbank-pay-sdk-android] | Beta |
+
+[swedbank-pay-sdk-php]: https://github.com/SwedbankPay/swedbank-pay-sdk-php
+[swedbank-pay-sdk-dotnet]: https://github.com/SwedbankPay/swedbank-pay-sdk-dotnet
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[final-definition] Move definitions to the end of the file (after the node at line `47`)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5308336)